Final Project/Server/fiimentor/controllers/professorController.ts

```diff
@@ -0,0 +1,52 @@
+import { ProfessorRepository } from "../Repositories/ProfessorRepository";
+import { UserRepository } from "../Repositories/UserRepository";
+import HttpStatus from "http-status-codes";
+
+interface ProfessorPayload {
+    id : number ,
+    UserId : number ,
+    FirstName : string,
+    LastName : string 
+}
+
+async function professor_list (req : any , res: any) {
```

Comment:
pune in denumire la functii un verb gen getProfessorList si respecta acelasi format camel case sau pascal case